PMSCore/UserProperty.cs

```diff
@@ -4,10 +4,12 @@ namespace PMSCore;
 
 public enum UserProperty
 {
-    [Description ("Gives a permission to a user")]
+    [Description("Gives a permission to a user")]
```

Comment:
duplicate distribution attribute, only 1 is needed to avoid possible errors

Reply:
Not duplicates